plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
On Arch Linux, the only package providing `sh` is `bash`, so it is always available. I think I'll stick with the small bash-isms.